t/02-pod.t

```diff
@@ -1,8 +1,8 @@
 use Test::Most;
 use Test::Warnings qw(:no_end_test :report_warnings);
+use Feature::Compat::Try;
 # no OpenQA::Test::TimeLimit for this trivial test
 
-eval 'use Test::Pod';
-plan skip_all => "Test::Pod required for testing POD" if $@;
+eval 'use Test::Pod; 1' or plan skip_all => "Test::Pod required for testing POD";
 
-all_pod_files_ok();
+Test::Pod::all_pod_files_ok();
```

Comment:
If you use `use` above, then the function is imported, so no need to use the full name:
```suggestion
all_pod_files_ok();
```